### PR TITLE
[LOG2LINES] Compile the tool for MSVC builds

### PR DESCRIPTION
**Pull request by learn-more**
This allows log2lines-like functionality for MSVC builds later on.

### CODE REVIEW

File: sdk/tools/log2lines/options.c [hunk 1]
```diff
@@ -17,6 +17,10 @@
 #include "log2lines.h"
 #include "options.h"
 
+#ifndef PATH_MAX
+#define PATH_MAX 260
+#endif
```
Comment (HBelusca) [inline sdk/tools/log2lines/options.c:22]: Could this go into log2lines/compat.h ? :thinking:

File: sdk/tools/rsym/rsym.h [hunk 2]
```diff
@@ -26,7 +26,7 @@ typedef struct {
   USHORT s_nlnno;    /* number of line number entries    */
   ULONG  s_flags;    /* flags                            */
 } SCNHDR;
-#pragma pack(4)
+#pragma pack(push, 4)
```
Comment (HBelusca) [inline sdk/tools/rsym/rsym.h:29]: Shouldn't the two structures above pragma-pack-push-4'ed too?

Comment (learn-more) [reply sdk/tools/rsym/rsym.h:29]: Don't know, I simply fixed the bug that it was not properly restored....

**Comment by binarymaster** [comment]
It seems this change breaks ARM64 builder :sweat_smile: 
```
D:\a\reactos\reactos\src\dll\win32\dbghelp\compat.h(554): fatal error C1189: #error:  "Unknown target platform"
```

**Comment by binarymaster** [comment]
@DarkFire01 maybe you have a patch at a hand for this one.

**Comment by binarymaster** [comment]
ARM64 quick fix PR is now merged, this one needs to be rebased on top of master to get fresh CI results.

**Review by HBelusca — APPROVED** [review]

**Review by binarymaster — APPROVED** [review]

**Review by HBelusca — APPROVED** [review]

**Pull request merged**